interface/forms/LBF/new.php

```diff
@@ -321,7 +321,7 @@ function end_row()
 ?>
 <html>
 <head>
-    <?php Header::setupHeader(['opener', 'common', 'datetime-picker', 'jquery-ui',]); ?>
+    <?php Header::setupHeader(['opener', 'common', 'datetime-picker']); ?>
 
     <style>
 
```

Comment:
On line 371 - Remove language attribute from script tag as it is no longer needed ([refer to styling guide](https://github.com/openemr/openemr/issues/3160#issuecomment-628259912))